src/modules/Dashboard/features/Participant/Assignments/AssignmentsTab/AssignmentsTab.hooks.tsx

```diff
@@ -207,7 +208,7 @@ export const useAssignmentsTab = ({
       );
       const isAssignDisplayed =
         canAssign && (!autoAssign || status === ActivityAssignmentStatus.Hidden);
-      const isAssignDisabled = !isAssignable || isTargetTeamMember;
+      const isAssignDisabled = !isAssignable || isTargetTeamMember || isLimitedRespondent;
```

Comment:
It's currently possible to access the assign menu item from the participants table more menu on a limited account. When this happens, the participant is pre-filled as the target of the assignment.

<img width="360" alt="image" src="https://github.com/user-attachments/assets/0664ba2d-b0b5-49d9-baf0-74adb585ae43"> <img width="360" alt="image" src="https://github.com/user-attachments/assets/db63e116-7880-427d-901d-67ef5183ad3b">


I think it would make sense to take a similar approach on at least the "about participant" view of the assignments tab. Though if we just did the same thing for both views, that'd also be fine.

Reply:
Good catch. That's actually a regression caused by incorrect logic for determining if the respondent is a limited account. What you're describing is actually part of the original AC for [M2-7828](https://mindlogger.atlassian.net/browse/M2-7828).

In fixing that bug I noticed another bug that would mislead users into thinking activities/flows could hypothetically be assigned to a limited account as a respondent; see explanation in [this commit](https://github.com/ChildMindInstitute/mindlogger-admin/pull/1974/commits/1221808051864316a18243ea245b401fa6b5a378).